docs/tutorials/scheduling.rst

```diff
@@ -0,0 +1,269 @@
+
+    >>> seq_scheduler = SequentialScheduler(constraints = global_constraints,
+    ...                                     observer = apo,
+    ...                                     transitioner = transitioner)
+    >>> schedule = Schedule(start_time, end_time)
+
+    >>> sequential_schedule = seq_scheduler(blocks, schedule)
+
+The second is a priority scheduler. It sorts the blocks by their
+priority (multiple blocks with the same priority will stay in the
+order they were in), then schedules them one-by-one at the best
+time for that block (highest score).
+
+.. code-block:: python
+
+    >>> from astroplan.scheduling import PriorityScheduler
+
+    >>> prior_scheduler = PriorityScheduler(constraints = global_constraints,
+    ...                                     observer = apo,
+    ...                                     transitioner = transitioner)
+    >>> schedule = Schedule(start_time, end_time)
+
+    >>> priority_schedule = prior_scheduler(blocks, schedule)
+
+Now that you have a schedule there are a few ways of viewing it.
+One way is to have it print a table where you can show, or hide,
+unused time and transitions with ``show_transitions`` and
+``show_unused`` (default is showing transitions and not unused).
+
+.. code-block:: python
+
+    >>> sequential_schedule.to_table()
+
+The other way is to plot the schedule against the airmass of the
+targets.
+
+.. code-block:: python
+
+    >>> from astroplan.plots import plot_schedule_airmass
+    >>> import matplotlib.pyplot as plt
+
+    >>> plt.figure(figsize = (14,6))
+    >>> plot_schedule_airmass(priority_schedule)
+    >>> plt.legend(loc = 1)
+    >>> plt.show()
+
+.. plot::
+
+    # first import everything we will need for the scheduling
+    import astropy.units as u
+    from astropy.time import Time
+    from astroplan import (Observer, FixedTarget, ObservingBlock, Transitioner, PriorityScheduler,
+                           Schedule)
+    from astroplan.constraints import AtNightConstraint, AirmassConstraint, MoonIlluminationConstraint
+    from astroplan.plots import plot_schedule_airmass
+    import matplotlib.pyplot as plt
+
+    # Now we define the targets, start time, end time, and observer.
+    Deneb = FixedTarget.from_name('deneb')
+    M13 = FixedTarget.from_name('m13')
+
+    start_time = Time('2016-06-07 02:00')
+    end_time = Time('2016-06-07 08:00')
+    apo = Observer.at_site('apo')
+
+    # Then define the constraints (global and specific) and make a list of the
+    # observing blocks that you want scheduled
+    global_constraints = [AirmassConstraint(max = 3, boolean_constraint = False),
+                          AtNightConstraint()]
+    rot = 20 * u.second
+    blocks = []
+    for filter in ['B', 'G', 'R']:
+        if filter == 'B':
+            constraints = [MoonIlluminationConstraint(max = 0.25)]
+        else:
+            constraints = None
+        blocks.append(ObservingBlock.from_exposures(Deneb, 1, 60*u.second, 17, rot,
```

Comment:
I think both targets need the same priority because we want to observe the target _and_ the standard.